packages/zoe/src/contracts/simpleExchange.js

```diff
@@ -6,7 +6,7 @@ import '../../exported';
 import {
   swap,
   satisfies,
-  checkIfProposal,
+  assertProposalShape,
   assertIssuerKeywords,
 } from '../contractSupport/zoeHelpers';
 
```

Comment:
Line 66 below refers to "the handle for the matched offer".

Reply:
ah, yep, I will update